src/mmw/mmw/settings/base.py

```diff
@@ -252,6 +252,13 @@ def get_env_setting(setting):
 # END URL CONFIGURATION
 
 
+# TILER
+MMW_TILER_HOST = environ.get('MMW_TILER_HOST', environ.get('MMW_TILER_IP', 'localhost'))  # noqa
```

Comment:
The reason being that these environmental variables should be set during provisioning and it isn't worth it to make these too robust.